types/block_test.go

```diff
@@ -151,10 +150,10 @@ func TestEquals(t *testing.T) {
 	var n1 Uint64 = 1234
 	var n2 Uint64 = 9876
 
-	b1 := &Block{Parents: NewTipSetKey(c1), Nonce: n1}
-	b2 := &Block{Parents: NewTipSetKey(c1), Nonce: n1}
-	b3 := &Block{Parents: NewTipSetKey(c1), Nonce: n2}
-	b4 := &Block{Parents: NewTipSetKey(c2), Nonce: n1}
```

Comment:
This is a weak test, but you'll need to do something to restore it. I suggest setting (at least) the state root in place of the nonce.